unittests/producer_schedule_tests.cpp

```diff
@@ -488,7 +488,7 @@ BOOST_FIXTURE_TEST_CASE( producer_one_of_n_test, validating_tester ) try {
    BOOST_REQUIRE_EQUAL( validate(), true );
 } FC_LOG_AND_RETHROW()
 
-BOOST_FIXTURE_TEST_CASE( producer_m_of_n_test, validating_tester ) try {
+BOOST_FIXTURE_TEST_CASE( producer_m_of_n_test, legacy_validating_tester ) try {
```

Comment:
Some of these tests that are not in `producer_schedule_if_tests` could/should be run in both legacy and savanna.